ide/xml.wsdl.model/test/unit/src/org/netbeans/modules/xml/wsdl/validator/WSDLSchemaValidatorTest.java

```diff
@@ -167,7 +168,13 @@ private void validate(URI uri, int expectedErrorCount)
         assertNotNull(vr.getValidationResult());
         
         ValidationHelper.dumpErrors(vr);
-        assertTrue("expect error " + expectedErrorCount,  vr.getValidationResult().size() == expectedErrorCount);
+        
+        // some missing element errors appear twice on newer JDKs
```

Comment:
Could reproduce the problem in JDK 19 (17 was ok).

Reply:
might depend on the update release version since it is also reproducible on JDK 11 with the zulu distribution:
https://github.com/apache/netbeans/actions/runs/3761176522/jobs/6394078954#step:94:2272
```
     [junit] Testcase: testValidateMessageNoName(org.netbeans.modules.xml.wsdl.validator.WSDLSchemaValidatorTest):	FAILED
    [junit] expect error 2
    [junit] junit.framework.AssertionFailedError: expect error 2
    [junit] 	at org.netbeans.modules.xml.wsdl.validator.WSDLSchemaValidatorTest.validate(WSDLSchemaValidatorTest.java:170)

```

i vaguely remember that xml libs got an update in OpenJDK, I just don't know the backporting rules for things like that.